### DISCUSSION TITLE
Installing on Ubuntu MATE

### DISCUSSION THREAD

**Discussion by nwb3** [MCPI-Devs/MCPIL#19] [Q&A]
Hello I have a raspberry pi 4 running Ubuntu MATE. I was just wondering if the install process was the same or if there were any other steps for the ubutnu install. Thanks!

**Comment by torralbaa** [comment]
Hello @nwb3!

The process _should_ be the same, since MCPIL only depends on TKinter for the GUI, and it's bundled/avaiable in almost every distro with any desktop environment.

However, please use [MCPIL-R](https://github.com/MCPI-Devs/MCPIL-R) or [MCPI-Docker](https://gitea.thebrokenrail.com/TheBrokenRail/minecraft-pi-docker) directly, since this branch of MCPIL currently is not being mantained and may not work for you.

While MCPIL-R may not support some (unmaintained) mods, it is my personal recommendation, since it provides an easy installation method and really good integration with MCPI-Docker.

Anyways, thanks for your interest in the project!

**Reply by nwb3** [reply]
@Alvarito050506 I installed through docker and received the following in the log files:

[main.log](https://github.com/MCPI-Devs/MCPIL/files/5758341/main.log)
[virgl.log](https://github.com/MCPI-Devs/MCPIL/files/5758342/virgl.log)

What happened was I opened it and the check boxes appeared  but after hitting launch it crashed
